test/test.js

```diff
@@ -1322,3 +1322,31 @@ ruleTester.run('generated-flow-types', rules['generated-flow-types'], {
     }
   ]
 });
+
+const FUTURE_ADDED_VALUE_MESSAGE =
+  "Do not use `'%future added value'`. It represents any potential " +
+  'value that the server might return in the future that the code ' +
+  'should handle.';
+ruleTester.run('no-future-added-value', rules['no-future-added-value'], {
+  valid: valid.concat([`const response: 'YES' | 'NO' = 'YES';`]),
+  invalid: [
+    {
+      // value location
+      code: `const response: 'YES' | 'NO' = '%future added value';`,
```

Comment:
Shouldn't this already fail just because of `flow` or am I mis-understanding some semantics of '%' in a string 😕

Reply:
`%` doesn't have any special semantics. I added it originally to make it look like something you're not supposed to use. Obviously that's not enough to discourage its use…

Reply:
This always reminds me of https://github.com/facebook/react/blob/80bff5397bf854750dbe7c286f61654ea58938c5/src/umd/ReactUMDEntry.js#L21-L22

😂